db/migrate/20241028193644_remove_can_manage_inventory_permission.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveCanManageInventoryPermission < ActiveRecord::Migration[7.0]
+  def up
+    safety_assured do
```

Comment:
I think that we don't have to worry about this warning from StrongMigrations since we weren't reading this attribute anywhere in the previous version of the code?

```
Caused by:
StrongMigrations::UnsafeMigration:
=== Dangerous operation detected #strong_migrations ===

Active Record caches attributes, which causes problems
when removing columns. Be sure to ignore the column:

class HmisRole < ApplicationRecord
  self.ignored_columns += ["can_manage_inventory"]
end

Deploy the code, then wrap this step in a safety_assured { ... } block.

class RemoveCanManageInventoryPermission < ActiveRecord::Migration[7.0]
  def change
    safety_assured { remove_column :hmis_roles, :can_manage_inventory }
  end
end
```

Reply:
Yeah that seems fine to me